File: 45848685 - UIDocumentBrowserViewController additional navigation bar button items ignore accessibility label/ExampleApp/Document.m
//
//  Document.m
//  ExampleApp
//
//  Created by Douglas Hill on 06/11/2018.
//  Copyright © 2018 Example. All rights reserved.
//

#import "Document.h"

@implementation Document
    
- (id)contentsForType:(NSString*)typeName error:(NSError **)errorPtr {
    // Encode your document with an instance of NSData or NSFileWrapper
    return [[NSData alloc] init];
}
    
- (BOOL)loadFromContents:(id)contents ofType:(NSString *)typeName error:(NSError **)errorPtr {
    // Load your document from contents
    return YES;
}

@end

File: 36129851 - UIDocumentBrowserViewController does not work in the Simulator/DocumentTestApp/Document.swift
//
//  Document.swift
//  DocumentTestApp
//
//  Created by Peter Steinberger on 19.12.17.
//  Copyright © 2017 Peter Steinberger. All rights reserved.
//

import UIKit

class Document: UIDocument {
    
    override func contents(forType typeName: String) throws -> Any {
        // Encode your document with an instance of NSData or NSFileWrapper
        return Data()
    }
    
    override func load(fromContents contents: Any, ofType typeName: String?) throws {
        // Load your document from contents
    }
}

